http.cc

```diff
@@ -106,6 +105,10 @@ static void handle_event(void *context, void *data, int len)
     }
 }
 
+static void handle_lost(h2o_tracer_t *tracer, uint64_t lost) {
+    fprintf(stderr, "Possibly lost %" PRIu64 " events\n", lost);
```

Comment:
The original message is `"Possibly lost %d samples"`, but I've changed a message here to identify easily.